tests/Unit/Routes/WebRoutesTest.php

```diff
@@ -210,4 +210,9 @@ public function testCentreRegistrationsSummaryGate()
             ->assertResponseOK()
         ;
     }
+
+    /** @test */
+    public function testRegistrationFamilyUpdateGate()
```

Comment:
This just returns true right now, correct?

Reply:
An unauthed user will be blocked blocked. An auth'd user will be waved through.

Reply:
Of course.